expression/schema.go

```diff
@@ -237,6 +238,17 @@ func (s *Schema) MemoryUsage() (sum int64) {
 	return
 }
 
+// GetExtraHandleColumn gets the extra handle column.
+func (s *Schema) GetExtraHandleColumn() *Column {
+	columnLen := len(s.Columns)
+	if columnLen > 0 && s.Columns[columnLen-1].ID == model.ExtraHandleID {
+		return s.Columns[columnLen-1]
+	} else if columnLen > 1 && s.Columns[columnLen-2].ID == model.ExtraHandleID {
+		return s.Columns[columnLen-2]
+	}
```

Comment:
I think it's better to find from the last to the first column to defend against any future changes.

Reply:
I thought about it, but there could be performance downgrade if it's a long Columns slice and no handleID is found.